src/main/resources/application.yml

```diff
@@ -0,0 +1,2 @@
+server:
+  port: 8080
```

Comment:
it's default value, remove it

Reply:
don't agree, it should be there to be self documented (to know which port will be used)

Reply:
so, why you override only port with default value, what about all other props?

Reply:
Port in application.yml used for development.
Port in docker container with web service should be 80, it's defacto standard.